src/node_errors.h

```diff
@@ -31,7 +32,6 @@ namespace node {
   V(ERR_SCRIPT_EXECUTION_INTERRUPTED, Error)                                 \
   V(ERR_SCRIPT_EXECUTION_TIMEOUT, Error)                                     \
   V(ERR_STRING_TOO_LONG, Error)                                              \
```

Comment:
Doesn't the `\` at the end of this line need to be removed now?

Reply:
We actually try to keep those, so that adding/removing lines has lower overhead, i.e. the previous style was actually the less common one.